packages/suite/src/views/onboarding/steps/Backup/index.tsx

```diff
@@ -108,6 +108,15 @@ const BackupStep = (props: Props) => {
                         <Translation id="TR_SKIP_SECURITY" />
```

Comment:
if the newly added block of code does the same thing as the already existing one, I would rather move the condition into it and just change Translation id

```
 status === 'finished' ? <Translation id="TR_SKIP_SECURITY_PIN" /> :  <Translation id="TR_SKIP_SECURITY" />
```

Or am I missing something?